src/lib.rs

```diff
@@ -1660,4 +1697,20 @@ mod tests {
         v.push(true);
         assert_eq!(v.vec.len(), 1);
     }
+
+    #[test]
+    fn to_bytes() {
+        let mut x = Vob::from_elem(3, true);
+        x.set(1, false);
+        assert_eq!(x.to_bytes(), [0b10100000]);
+    }
+
+    #[test]
+    fn to_bytes_larger(){
+        let mut x = Vob::from_elem(9, false);
+        x.set(2, true);
+        x.set(8, true);
+        assert_eq!(x.to_bytes(), [0b00100000, 0b10000000]);
+    }
```

Comment:
I would like to see a test that shows exactly 8 elements and a test for 0 elements.